packages/cli/src/commands/init.ts

```diff
@@ -47,6 +47,9 @@ export const initCommand = async (options: InitCommandOptions) => {
   // Detect if are are in a Next.js project
   const isNextJsProject = await detectNextJsProject(resolvedPath);
 
+  //Detect Next.Js version
+  const nextJsVersion = await detectNextJsVersion(resolvedPath)
+
```

Comment:
I think it'd be better if this code was after the check for if it's a Next.js project, so moved to line 60.